source/tests/test_neighbor_stat.py

```diff
@@ -0,0 +1,47 @@
+import shutil
+import numpy as np
+import unittest
+import dpdata
+
+from deepmd.entrypoints.neighbor_stat import neighbor_stat
+
+def gen_sys(nframes):
+    natoms = 1000
+    data = {}
+    X, Y, Z = np.mgrid[0:9:10j, 0:9:10j, 0:9:10j]
+    positions = np.vstack([X.ravel(), Y.ravel(), Z.ravel()]).T #+ 0.1
+    data['coords'] = np.repeat(positions[np.newaxis, :, :], nframes, axis=0)
+    data['forces'] = np.random.random([nframes, natoms, 3])
+    data['cells'] = np.array([10., 0., 0., 0., 10., 0., 0., 0., 10.]).reshape(1,3,3)
+    data['energies'] = np.random.random([nframes, 1])
+    data['atom_names'] = ['TYPE']
+    data['atom_numbs'] = [1000]
+    data['atom_types'] = np.repeat(0, 1000)
+    return data
+
+
+class TestNeighborStat(unittest.TestCase):
+    def setUp(self):
+        data0 = gen_sys(1)
+        sys0 = dpdata.LabeledSystem()
+        sys0.data = data0
+        sys0.to_deepmd_npy('system_0', set_size = 10)
+        
+    def tearDown(self):
+        shutil.rmtree('system_0')
+
+    def test_neighbor_stat(self):
+        # set rcut to 0. will cause a core dumped
+        # TODO: check what is wrong
+        for rcut in (3., 6., 11.):
```

Comment:
@denghuilu setting `rcut` to `0.` will cause "core dumped". Any ideas?

Reply:
(Although, I don't think some one will set it to 0)

Reply:
It might be caused by this line: https://github.com/deepmodeling/deepmd-kit/blob/159e45d248b0429844fb6a8cb3b3a201987c8d79/source/lib/src/neighbor_list.cc#L665